packages/create-react-native-library/templates/example/example/windows/{%- project.name %}Example/App.xaml.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.ReactNative;
+using Windows.ApplicationModel.Activation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace <%- project.name %>Example
+{
+    sealed partial class App : ReactApplication
+    {
+        public App()
+        {
+#if BUNDLE
+            JavaScriptBundleFile = "index.windows";
+            InstanceSettings.UseWebDebugger = false;
+            InstanceSettings.UseFastRefresh = false;
+#else
+            JavaScriptMainModuleName = "index";
+            InstanceSettings.UseWebDebugger = true;
+            InstanceSettings.UseFastRefresh = true;
+#endif
+
+#if DEBUG
+            InstanceSettings.UseDeveloperSupport = true;
+#else
+            InstanceSettings.UseDeveloperSupport = false;
+#endif
+
+            Microsoft.ReactNative.Managed.AutolinkedNativeModules.RegisterAutolinkedNativeModulePackages(PackageProviders); // Includes any autolinked modules
+
+            PackageProviders.Add(new Microsoft.ReactNative.Managed.ReactPackageProvider());
```

Comment:
I think we recently deleted this line (this past week in fact :) )

Reply:
This template is for late of v0.63, so I will need to keep this line, but once the whole Bob is updated, I'll go with removing it 👍